### ISSUE TITLE
[system/uri] read_routes() is not working in CWP7

### ISSUE THREAD

**Issue by gnh1201** [gnh1201/reasonableframework#142]
## Summary
When ReasonableFramework is installed on CWP7, the `read_routes()` function returns an empty array.

I will investigate this issue because it was confirmed to work properly on CWP6 previously.

## Related links
* https://centos-webpanel.com/

**Comment by gnh1201** [comment]
## dumped result of read_requests()

```
array(20) {
  ["_ALL"]=>
  array(0) {
  }
  ["_POST"]=>
  array(0) {
  }
  ["_GET"]=>
  array(0) {
  }
  ["_URI"]=>
  string(10) "/extlist/8"
  ["_FILES"]=>
  array(0) {
  }
  ["_RAW"]=>
  string(0) ""
  ["_JSON"]=>
  bool(false)
  ["_SEAL"]=>
  bool(false)
  ["_CSPT"]=>
  bool(false)
  ["_SERVER"]=>
  array(38) {
    ["CONTEXT_DOCUMENT_ROOT"]=>
    string(26) "/home/utilhome/public_html"
    ["CONTEXT_PREFIX"]=>
    string(0) ""
    ["DOCUMENT_ROOT"]=>
    string(26) "/home/utilhome/public_html"
    ["GATEWAY_INTERFACE"]=>
    string(7) "CGI/1.1"
    ["HTTP_ACCEPT"]=>
    string(135) "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7"
    ["HTTP_ACCEPT_ENCODING"]=>
    string(13) "gzip, deflate"
    ["HTTP_ACCEPT_LANGUAGE"]=>
    string(2) "ko"
    ["HTTP_CONNECTION"]=>
    string(10) "keep-alive"
    ["HTTP_COOKIE"]=>
    string(36) "PHPSESSID=rr1pdcgkv318oq7padcfrr54q4"
    ["HTTP_HOST"]=>
    string(7) "exts.kr"
    ["HTTP_UPGRADE_INSECURE_REQUESTS"]=>
    string(1) "1"
    ["HTTP_USER_AGENT"]=>
    string(125) "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 Edg/124.0.0.0"
    ["PATH"]=>
    string(32) "/bin:/usr/bin:/usr/local/php/bin"
    ["PHPRC"]=>
    string(14) "/home/utilhome"
    ["QUERY_STRING"]=>
    string(0) ""
    ["REDIRECT_STATUS"]=>
    string(3) "200"
    ["REDIRECT_UNIQUE_ID"]=>
    string(27) "Zi-gkpd9XXzAyxgTQB0sOQAAAIM"
    ["REDIRECT_URL"]=>
    string(10) "/extlist/8"
    ["REMOTE_ADDR"]=>
    string(9) "10.8.0.16"
    ["REMOTE_PORT"]=>
    string(5) "51323"
    ["REQUEST_METHOD"]=>
    string(3) "GET"
    ["REQUEST_SCHEME"]=>
    string(4) "http"
    ["REQUEST_URI"]=>
    string(10) "/extlist/8"
    ["SCRIPT_FILENAME"]=>
    string(36) "/home/utilhome/public_html/index.php"
    ["SCRIPT_NAME"]=>
    string(10) "/index.php"
    ["SERVER_ADDR"]=>
    string(9) "10.8.0.12"
    ["SERVER_ADMIN"]=>
    string(17) "webmaster@exts.kr"
    ["SERVER_NAME"]=>
    string(7) "exts.kr"
    ["SERVER_PORT"]=>
    string(2) "80"
    ["SERVER_PROTOCOL"]=>
    string(8) "HTTP/1.1"
    ["SERVER_SIGNATURE"]=>
    string(0) ""
    ["SERVER_SOFTWARE"]=>
    string(35) "Apache/2.4.57 (Unix) OpenSSL/1.1.1k"
    ["UNIQUE_ID"]=>
    string(27) "Zi-gkpd9XXzAyxgTQB0sOQAAAIM"
    ["ORIG_PATH_INFO"]=>
    string(10) "/extlist/8"
    ["ORIG_PATH_TRANSLATED"]=>
    string(36) "/home/utilhome/public_html/index.php"
    ["PHP_SELF"]=>
    string(10) "/index.php"
    ["REQUEST_TIME_FLOAT"]=>
    float(1714397331.461)
    ["REQUEST_TIME"]=>
    int(1714397331)
  }
  ["_HEADER"]=>
  array(8) {
    ["Accept"]=>
    string(135) "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7"
    ["Accept-Encoding"]=>
    string(13) "gzip, deflate"
    ["Accept-Language"]=>
    string(2) "ko"
    ["Connection"]=>
    string(10) "keep-alive"
    ["Cookie"]=>
    string(36) "PHPSESSID=rr1pdcgkv318oq7padcfrr54q4"
    ["Host"]=>
    string(7) "exts.kr"
    ["Upgrade-Insecure-Requests"]=>
    string(1) "1"
    ["User-Agent"]=>
    string(125) "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 Edg/124.0.0.0"
  }
  ["all"]=>
  array(0) {
  }
  ["post"]=>
  array(0) {
  }
  ["get"]=>
  array(0) {
  }
  ["uri"]=>
  string(10) "/extlist/8"
  ["files"]=>
  array(0) {
  }
  ["raw"]=>
  string(0) ""
  ["json"]=>
  bool(false)
  ["seal"]=>
  bool(false)
  ["cspt"]=>
  bool(false)
}
```

## dumped result of read_routes()

```
array(0) {
}
```

**Comment by gnh1201** [comment]
This issue has been resolved in issue #138, and it has been confirmed that the system where the problem occurred does not have the relevant changes applied.

Here is the link associated with this issue:

* #140
* #138
